### PR TITLE
Safety debug testing

### PR DESCRIPTION
**Pull request by scholvat**

### CODE REVIEW

**Comment by scholvat** [comment]
Swapped TX and RX pins

File: Safety/Src/main.c [hunk 1]
```diff
@@ -114,7 +114,7 @@ int main(void)
   HAL_GPIO_WritePin(LED1_GPIO_Port, LED1_Pin, GPIO_PIN_SET);
   HAL_GPIO_WritePin(LED2_GPIO_Port, LED2_Pin, GPIO_PIN_SET);
   HAL_GPIO_WritePin(LED3_GPIO_Port, LED3_Pin, GPIO_PIN_SET);
-
+
```
Comment (ianjfrosst) [inline Safety/Src/main.c]: Lame

Comment (edenfrosst) [inline Safety/Src/main.c]: Lame

**Review by edenfrosst — CHANGES_REQUESTED** [review]

File: Safety/Src/main.c (hunk 1 above)
Comment (scholvat) [reply Safety/Src/main.c]: Better? :P

**Review by edenfrosst — APPROVED** [review]

**Pull request merged**